README-spec.md

```diff
@@ -79,7 +79,42 @@ with this Batch are written to the database.
 
 ## Admin Dashboard
 
-This will be spec'ed later. For now, all tables should be visible.
+There are a list of different experiments. (In the first version, we will only have one experiment.) If you click on one, you go to the Admin Experiment View.
+There is a link to the Admin User View.
+
+There should be a simple link or something that allows me to view all raw tables.
+
+### Admin User View
+
+There is a table of annotators.
+
+There is a field to add their hourly rate as a float. (This should be added to the User model.)
+
+The columns are: email, experiment name, number of tasks completed, percent of gold correct, interannotator agreement, ROI, lock?, delete?
+
```

Comment:
Lock and delete should be an option, since they're tasks that can only be performed by admin users.

Reply:
@DeFidelity I don't understand. This is for the admin interface ONLY. What do you mean

Reply:
I'm addressing the "?" You put after both of them, and what I mean is that it should be possible for admin to lock users or delete them right from the admin interface.

Reply:
Ah that was just to suggest the text should include a question mark in the table name but maybe it's unnecessary.

Reply:
I'm not clear whether the columns mentioned above are of annotators table.